spec/search_builders/hyrax/dashboard/collections_search_builder_spec.rb

```diff
@@ -55,6 +55,11 @@
 
     subject { builder.gated_discovery_filters }
 
+    before do
```

Comment:
What problem is this addition addressing? This spec passes locally for me without manually saving the users.

Reply:
I was getting the following error at the .to_sippity_agent line: Failure/Error: create_sipity_agent! ActiveRecord::RecordNotSaved: You cannot call create unless the parent is saved
I can take those saves out and retry it locally.